README.asciidoc

```diff
@@ -958,7 +958,7 @@ Some options are built in Kakoune, and can be used to control its behaviour:
    - `option=<opt-name>` where <opt-name> is a _completions_ option.
  * `static_words` _str-list_: list of words that are always added to completion
      candidates when completing words in insert mode.
- * `completions_extra_word_chars` _str_: a string containing all additional character
```

Comment:
>you wont find codepoint-list directly, its a Vector<Codepoint> option

That's my point, if it's not a declarable/complete-able type, then to the users it doesn't exist, so it will be confusing to see its name appear in the documentation.